### PR TITLE
Upgrade the migration page, include VCL version table

### PR DESCRIPTION
**Pull request by hermunn**
This should close issue 3065 in Varnish Cache.

### CODE REVIEW

File: R1/source/tips/migration/index.rst [hunk 1]
```diff
@@ -3,10 +3,56 @@
...
+Upgrading from 4.x to the latest LTS should be very straightforward,
+and require much less effort than going to 3.x to 4.x.
+
+Versions of the VCL language
+============================
+
+Currently there are two official versions of the VCL language, 4.0 and
+4.1, and they are only slightly different.
+
+This table shows which versions of the VCL language is supported in
```
Comment (nigoroll) [inline R1/source/tips/migration/index.rst]: *are* supported

File: R1/source/tips/migration/index.rst [hunk 2]
```diff
@@ -3,10 +3,56 @@
...
-* `Migrating to 4.1.x
-  </docs/4.1/whats-new/upgrading.html>`_
+* `Migrating to 4.1.x </docs/4.1/whats-new/upgrading.html>`_
+* `Upgrading to 5.0 </docs/6.0/whats-new/upgrading-5.0.html>`_
+* `Upgrading to 5.1 </docs/6.0/whats-new/upgrading-5.1.html>`_
+* `Upgrading to 5.2 </docs/6.0/whats-new/upgrading-5.2.html>`_
+* `Upgrading to 6.0 </docs/6.0/whats-new/upgrading-6.0.html>`_
+
+Releases after 6.0 do not come with Long Time Support, but are
+released as "fresh". A "fresh" release will get security update for a
```
Comment (nigoroll) [inline R1/source/tips/migration/index.rst]: update *s*

**Comment by hermunn** [comment]
> Yes, but shouldn't much of this be in the reference manual instead?

I think that since this is about many releases, and that it will change as things go EOL, we do not want to change this in all of the branches. At least that is what I understood as the consensus at the time, when it was discussed during the bugwash. I will update the grammar problems @nigoroll pointed out and force push.

**Comment by hermunn** [comment]
Fixed typos and force pushed. @nigoroll , if you are happy you can press the Rebase and Merge button.

**Pull request merged**